### DISCUSSION TITLE
How to get the IP:Port info of che client?

### DISCUSSION THREAD

**Discussion by deahhh** [GenieFramework/Genie.jl#618] [Q&A] [answered]
Sometime, I want to construct a new connection between two clients.
If I know IP:Port info of them, I can send the info of one to the other.
And, send connection request from client through WebChannel.

**Comment by essenciary** [comment, marked as answer]
@deahhh So there's this `features_peerinfo` the `Configuration` module. It's disabled by default but if you enable it, in `params()` you'll get a new `:peer` entry that should return this info. It's not a very commonly used and tested feature so please let me know how it works.

**Reply by deahhh** [reply]
Thanks!
